custom_libs/subliminal_patch/providers/animekalesi.py

```diff
@@ -0,0 +1,332 @@
+
+                                # Find uploader information
+                                uploader = None
+                                translator_info = soup.find('strong', text='Altyazı/Çeviri:')
+                                if translator_info and translator_info.parent:
+                                    strong_tags = translator_info.parent.find_all('strong')
+                                    for i, tag in enumerate(strong_tags):
+                                        if tag.text == 'Altyazı/Çeviri:':
+                                            if i + 1 < len(strong_tags):
+                                                uploader = tag.next_sibling
+                                                if uploader:
+                                                    uploader = uploader.strip()
+                                            else:
+                                                uploader = tag.next_sibling
+                                                if uploader:
+                                                    uploader = uploader.strip()
+                                            break
+
+                                version = f"{series_data['title']} - S{current_season:02d}E{current_episode:02d}"
+                                if uploader:
+                                    version += f" by {uploader}"
+
+                                try:
+                                    subtitle = self.subtitle_class(
+                                        Language('tur'),
+                                        link_url,
+                                        series_data['title'],
+                                        current_season,
+                                        current_episode,
+                                        version,
+                                        download_link,
+                                        uploader=uploader
+                                    )
+                                    subtitle.release_group = None
+                                    subtitles.append(subtitle)
+                                except Exception as e:
+                                    logger.error('Error creating subtitle object: %s', e)
+                                    continue
+
+                        except Exception as e:
+                            logger.error('Error processing subtitle page %s: %s', link_url, e)
+                            continue
+
+        except Exception as e:
+            logger.error('Error querying subtitles: %s', e)
+
+        return subtitles
+
+    def list_subtitles(self, video: Episode, languages: Set[Language]) -> List[AnimeKalesiSubtitle]:
+        if not video.series or not video.episode:
+            return []
+
+        # Language check - both alpha3 and IETF codes
+        supported_languages = {l for l in languages if l.alpha3 == 'tur' or l.alpha2 == 'tr'}
```

Comment:
You shouldn't even get there if language is not Turkish. You'll get something like "no language to search for" in debug logging. No need to test for this here.